### ISSUE TITLE
tests are failing on ppc64

### ISSUE THREAD

**Issue by ignatenkobrain** [gimli-rs/addr2line#82]
```
failures:
---- identity_map stdout ----
	thread 'identity_map' panicked at 'assertion failed: `(left != right)`
  left: `0`,
 right: `0`', tests/integration_test.rs:76:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
---- with_functions stdout ----
	thread 'with_functions' panicked at 'assertion failed: `(left != right)`
  left: `0`,
 right: `0`', tests/integration_test.rs:148:4
failures:
    identity_map
    with_functions
```

Not sure how to debug it...

**Comment by philipc** [comment]
Looks like this is version 0.5.0, not git?

**Comment by ignatenkobrain** [comment]
@philipc yeah, it is..

**Comment by philipc** [comment]
I can reproduce the problem in travis CI. It looks like we're failing to symbolize any addresses at all. I don't want to spend the effort debugging this via travis CI though, because the code has changed a lot since version 0.5.0.

I tried reproducing this for the git version, but currently the cross compilation environment doesn't have clang, which is needed for bindgen for findshlibs. Are you able to test if the git version works?

**Comment by ignatenkobrain** [comment]
git version is fine. I  tried to bisect, but they are totally different versions...

**Comment by philipc** [comment]
Ok, thanks for testing that. Yes, it's totally different so it's not worth spending effort on 0.5.0.

**Comment by philipc** [comment]
Published 0.6.0. Please reopen if there are still problems.
